### PR TITLE
Simplify resource requirements for load test

### PR DESCRIPTION
**Pull request by wojtek-t**
This should help a bit with variance of pod-startup times - a bit more context can be found in:
https://github.com/kubernetes/perf-tests/issues/1311#issuecomment-1077405626

It also makes our configs a bit simpler.

/assign @mborsz

### CODE REVIEW

File: clusterloader2/testing/load/deployment.yaml [hunk 1]
```diff
@@ -1,6 +1,8 @@
-{{$HostNetworkMode := DefaultParam .CL2_USE_HOST_NETWORK_PODS false}}
-{{$CpuRequest := DefaultParam .CpuRequest "10m"}}
-{{$MemoryRequest := DefaultParam .MemoryRequest "10M"}}
+{{$HostNetworkMode := DefaultParam .CL2_USE_HOST_NETWORK_PODS false}}i
```
Comment (mborsz) [inline clusterloader2/testing/load/deployment.yaml:1]: "i" suffix?

Comment (wojtek-t) [reply clusterloader2/testing/load/deployment.yaml]: good catch - fixed

**Comment by wojtek-t** [comment]
@mborsz  - PTAL (I fixed the test by adding default values to simple-deployment.yaml)

**Pull request merged**